docs/intro/contributing.txt

```diff
@@ -327,42 +313,25 @@ Now for our hands-on example.
 
 __ https://en.wikipedia.org/wiki/Test-driven_development
 
-Writing some tests for ticket #24788
-------------------------------------
-
-Ticket :ticket:`24788` proposes a small feature addition: the ability to
-specify the class level attribute ``prefix`` on Form classes, so that::
-
-    […] forms which ship with apps could effectively namespace themselves such
-    that N overlapping form fields could be POSTed at once and resolved to the
-    correct form.
+Writing a test for ticket #99999
+--------------------------------
 
-In order to resolve this ticket, we'll add a ``prefix`` attribute to the
-``BaseForm`` class. When creating instances of this class, passing a prefix to
-the ``__init__()`` method will still set that prefix on the created instance.
-But not passing a prefix (or passing ``None``) will use the class-level prefix.
-Before we make those changes though, we're going to write a couple tests to
-verify that our modification functions correctly and continues to function
-correctly in the future.
+In order to resolve this ticket, we'll add a ``make_toast()`` function to the
+top-level ``django`` module. First we are going to write a test that tries to
+use the function and check that its output looks correct.
 
-Navigate to Django's ``tests/forms_tests/tests/`` folder and open the
-``test_forms.py`` file. Add the following code on line 1674 right before the
-``test_forms_with_null_boolean`` function::
+Navigate to Django's ``tests/shortcuts/`` folder and create a new file
+``test_make_toast.py``. Add the following code::
 
-    def test_class_prefix(self):
-        # Prefix can be also specified at the class level.
-        class Person(Form):
-            first_name = CharField()
-            prefix = 'foo'
+    from django.shortcuts import make_toast
 
-        p = Person()
-        self.assertEqual(p.prefix, 'foo')
 
-        p = Person(prefix='bar')
-        self.assertEqual(p.prefix, 'bar')
+    class MakeToastTests(SimpleTestCase):
```

Comment:
SImpleTestCase need to be imported

Reply:
fixed, and added `--cached` argument to `git diff` as you emailed me about